### DISCUSSION TITLE
moqi_wan_zrm和moqi_zrm的区别

### DISCUSSION THREAD

**Discussion by ZMProHelper** [gaboolic/rime-config#130] [Q&A] [answered]
'wan'表示带字库?
'moqi'表示使用的是moqi辅助码?
所以moqi_wan_zrm和moqi_zrm使用的辅助码都是moqi辅助码?

**Comment by gaboolic** [comment, marked as answer]
以前用的moqi_zrm
后来把各种双拼和辅助码用拼写规则统一了，是moqi_wan_zrm
两个都用的自然码辅助码  
moqi_wan_zrm可以修改辅助码

pro_comment_format:                   # 超级注释，子项配置 true 开启，false 关闭
  fuzhu_code: true                    # 启用辅助码提醒，用于辅助输入练习辅助码，成熟后可关闭
  candidate_length: 1                 # 候选词辅助码提醒的生效长度，0为关闭  但同时清空其它，应当使用上面开关来处理    
  fuzhu_type: zrm                     # 用于匹配对应的辅助码注释显示，可选显示类型有：moqi, flypy, zrm, jdh, cj, tiger, wubi,选择一个填入，应与上面辅助码类型一致

**Reply by ZMProHelper** [reply]
![image](https://github.com/user-attachments/assets/c22f57be-35f3-4c13-ac02-87ad3e307ba3)
想改成moqi辅助码,改aux_patch就足够了吗?
